samples/excel/85-preview-apis/culture-info-date-time.yaml

```diff
@@ -0,0 +1,138 @@
+            await context.sync();
+
+            // System settings are read-only and set in the user's operating system.
+            const systemLongDatePattern = context.application.cultureInfo.datetimeFormat.longDatePattern;
+            const systemShortDatePattern = context.application.cultureInfo.datetimeFormat.shortDatePattern;
+            const systemDateSeparator = context.application.cultureInfo.datetimeFormat.dateSeparator;
+            const systemLongTimePattern = context.application.cultureInfo.datetimeFormat.longTimePattern;
+            const systemTimeSeparator = context.application.cultureInfo.datetimeFormat.timeSeparator;
+
+            console.log("System date/time settings: ");
+            console.log(`  System long date format: ${systemLongDatePattern}`);
+            console.log(`  System short date format: ${systemShortDatePattern}`);
+            console.log(`  System date separator: ${systemDateSeparator}`);
+            console.log(`  System long time format: ${systemLongTimePattern}`);
+            console.log(`  System time separator: ${systemTimeSeparator}`);
+
+            await context.sync();
+          });
+        }
+
+        /* Use the cultural settings API to write date and time settings in your table. */
+        async function writeDateTimeSetting() {
+          await Excel.run(async (context) => {
+            context.application.cultureInfo.datetimeFormat.load([
+              "longDatePattern",
+              "shortDatePattern",
+              "dateSeparator",
+              "longTimePattern",
+              "timeSeparator"
+            ]);
+            await context.sync();
+
+            // System settings are read-only and set in the user's operating system.
+            const systemLongDatePattern = context.application.cultureInfo.datetimeFormat.longDatePattern;
+            const systemShortDatePattern = context.application.cultureInfo.datetimeFormat.shortDatePattern;
+            const systemDateSeparator = context.application.cultureInfo.datetimeFormat.dateSeparator;
+            const systemLongTimePattern = context.application.cultureInfo.datetimeFormat.longTimePattern;
+            const systemTimeSeparator = context.application.cultureInfo.datetimeFormat.timeSeparator;
+
+            const sheet = context.workbook.worksheets.getActiveWorksheet();
+            const longDate = sheet.getRange("A2:B2");
+            longDate.values = [["Long date", systemLongDatePattern]];
+            const shortDate = sheet.getRange("A3:B3");
+            shortDate.values = [["Short date", systemShortDatePattern]];
+            const shortDate = sheet.getRange("A4:B4");
+            shortDate.values = [["Date separator", systemDateSeparator]];
+            const shortDate = sheet.getRange("A5:B5");
+            shortDate.values = [["Long time format", systemLongTimePattern]];
+            const shortDate = sheet.getRange("A6:B6");
+            shortDate.values = [["Time separator", systemTimeSeparator]];
```

Comment:
While it doesn't matter at this scale, there is a best practice to reduce the number of Range objects created. Consider rewriting this to be one `getRange` operation and one set `values` operation.